src/cc/usdt.h

```diff
@@ -255,6 +259,7 @@ class Context {
   ino_t inode() const { return mount_ns_instance_->target_ino(); }
 
   Probe *get(const std::string &probe_name);
+  Probe *get(const std::string &provider_name, const std::string &probe_name);
```

Comment:
Can you also update C++ API to use the provider-and-name version of `get`? It currently only has the binary code path but we'll need to add PID code path soon:) And in that case maybe we don't need to keep the (kind of incorrect) `get(name)` anymore?

Reply:
Yes, for C++ API, USDT constructor needs provider + probe_name, so we can use `get(provider_, name_)` instead of `get(name_)`. Just updated the patch to reflect this.